src/plugins/workspace/public/components/workspace_form/workspace_use_case.tsx

```diff
@@ -54,21 +60,60 @@ const WorkspaceUseCaseCard = ({
   );
 };
 
+type AvailableUseCase = Pick<
+  WorkspaceUseCaseObject,
+  'id' | 'title' | 'description' | 'systematic'
```

Comment:
Maybe this is irrelevant, but what does `systematic` stand for? cc @wanglam

Reply:
For filtering out system nav group, we won't show system nav groups in the use case selector.